src/util/psbt/map/output.rs

```diff
@@ -106,11 +121,11 @@ impl TapTree {
     ///
     /// # Return
     /// A `TapTree` iff the `inner` builder is complete, otherwise return the inner as `Err`.
```

Comment:
The comment here was not updated to reflect the change

Reply:
Fixed